src/applications/mhv/secure-messaging/components/ComposeForm/ComposeForm.jsx

```diff
@@ -479,6 +479,7 @@ const ComposeForm = props => {
             }}
             status="warning"
             data-testid="quit-compose-double-dare"
+            data-dd-action-name="Save Error Modal"
```

Comment:
curious how this works on a modal? does it trigger an action that is reported back to DD RUM?